src/model/builtin_options_impl.rs

```diff
@@ -1,3 +1,4 @@
+
 use super::{BuiltinOptions, BuiltinOptionsUnion, NativeTable};
```

Comment:
Looks like this file had `rustfmt` run on it and that's why there's all the changes.

Reply:
The changes in these `_impl` files come from when we run with the `generate_model_apis` feature on.